src/oumi/datasets/grpo/rewards/__init__.py

```diff
@@ -26,4 +26,4 @@
     "compute_soft_target_token_length_reward",
     "compute_sharp_target_token_length_reward",
     "countdown_reward",
-]
+]
```

Comment:
```suggestion
]

```

nit: revert this change to keep the newline at end of file. Should fix the erroring CPU test